startup/GafferAppleseedUI/appleseedExamples.py

```diff
@@ -51,3 +52,28 @@
 	]
 )
 
+GafferUI.Examples.registerExample(
+	"Rendering/Multi-shot Render Spreadsheet",
+	"$GAFFER_ROOT/resources/examples/multiShotRenderSpreadsheet.gfr",
+	description = """
+	Demonstrates how to use the Spreadsheet node to vary renderer
+	settings per shot.
+	""",
+	notableNodes = [
+		Gaffer.Spreadsheet,
+		GafferAppleseed.AppleseedOptions
+	]
+)
+
+GafferUI.Examples.registerExample(
+	"Rendering/Per-location Light Tweak Spreadsheet",
+	"$GAFFER_ROOT/resources/examples/perLocationLightTweakSpreadsheet.gfr",
```

Comment:
Can we move the files into the matching folders?

Reply:
202ed056e12259507a6864b11bcfae6ad716bd16